gen3_util/projects/__init__.py

```diff
@@ -9,16 +9,16 @@
 
 class ProjectSummary(BaseModel):
     """Summary of a project."""
-    in_sheepdog: bool = False
-    """Project exists flag"""
+    exists: bool = False
+    """Project exists in sheepdog flag"""
     permissions: list[dict[str, Any]] = []
 
 
 class ProjectSummaries(BaseModel):
     """Summary of projects, including messages."""
     endpoint: str
     """The commons url"""
-    projects: dict[str, ProjectSummary]
+    projects: dict[str, ProjectSummary] | list[str] = {}
```

Comment:
This line was causing the below error:
```(venv) peterkor@RNB11238 gen3_util % gen3_util

/Users/peterkor/Library/Python/3.9/lib/python/site-packages/urllib3/__init__.py:34: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(


Traceback (most recent call last):
  File "/Users/peterkor/Library/Python/3.9/bin/gen3_util", line 33, in <module>
    sys.exit(load_entry_point('gen3-util', 'console_scripts', 'gen3_util')())
  File "/Users/peterkor/Library/Python/3.9/bin/gen3_util", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/peterkor/Desktop/gen3_util/gen3_util/cli/cli.py", line 13, in <module>
    from gen3_util.files.cli import file_group
  File "/Users/peterkor/Desktop/gen3_util/gen3_util/files/__init__.py", line 4, in <module>
    from gen3_util.projects import get_user
  File "/Users/peterkor/Desktop/gen3_util/gen3_util/projects/__init__.py", line 17, in <module>
    class ProjectSummaries(BaseModel):
  File "/Users/peterkor/Desktop/gen3_util/gen3_util/projects/__init__.py", line 21, in ProjectSummaries
    projects: dict[str, ProjectSummary] | list[str] = {}
TypeError: unsupported operand type(s) for |: 'types.GenericAlias' and 'types.GenericAlias'
```
resolved by changing it to ```projects: Union[dict[str, ProjectSummary], list[str]] = {}```
and importing Union from typing

Reply:
thanks.  fixed it